### PR TITLE
enhancements/monitoring/user-workload-monitoring: fix formatting, add details

### PR DESCRIPTION
**Pull request by s-urbaniak**
/cc @bwplotka

### CODE REVIEW

File: enhancements/monitoring/user-workload-monitoring.md [hunk 1]
```diff
@@ -128,33 +128,33 @@ As a member of the operations tenant, I’d like to make sure that one tenant do
...
-### US14
+#### US14
 
 As a member of the operations team, I'd like to give dedicated users the permission to enable/disable and configure the user workload monitoring stack.
 
-### US15
+#### US15
 
-As a member of the operations team I want to see all alerts (cluster and user workload alerts) in the admin perspective and can silence any of them.
+As a member of the operations team I want to see all alerts (cluster and user workload alerts) in the admin panel and can silence any of them.
```
Comment (bwplotka) [inline enhancements/monitoring/user-workload-monitoring.md]: Worth to mention all sources of those alerts?

Comment (bwplotka) [reply enhancements/monitoring/user-workload-monitoring.md]: for clarity?

File: enhancements/monitoring/user-workload-monitoring.md [hunk 2]
```diff
@@ -319,6 +319,7 @@ The following new named roles are deployed via cluster-monitoring-operator.
 
 #### `monitoring-rules-view`
 This is a cluster role which can be bound against a concrete namespace by the cluster admin.
```
Comment (bwplotka) [inline enhancements/monitoring/user-workload-monitoring.md]: newline here, not below ;p

Comment (s-urbaniak) [reply enhancements/monitoring/user-workload-monitoring.md]: :man_facepalming:

**Pull request closed without merging**